source/MaterialXRender/ShaderRenderer.h

```diff
@@ -3,11 +3,11 @@
 // All rights reserved.  See LICENSE.txt for license.
 //
 
-#ifndef MATERIALX_SHADERVALIDATOR_H
```

Comment:
Does this pull request need to be merged with the latest changes in MaterialX master?  I wouldn't expect to see references to shader validators here.

Reply:
I thought that I had done this already. Could be I didn't re-merge from master back to this branch.